addons/mail/models/mail_ice_server.py

```diff
@@ -42,6 +44,7 @@ def _get_ice_servers(self):
             (account_sid, auth_token) = get_twilio_credentials(self.env)
             if account_sid and auth_token:
                 url = f'https://api.twilio.com/2010-04-01/Accounts/{account_sid}/Tokens.json'
+                _logger.info("Contacting twilio at: %s", url)
```

Comment:
Why is there an extra log? That looks irrelevant to fixing the race condition.

Reply:
xdo wanted to check it during multi run test on runbot, I will remove it before merge